.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+### Issue details
```

Comment:
Perhaps start with a friendly intro?

`Thanks for wanting to report an issue you've found in node.js. Please fill in below template. If unsure about something, just do as best as you're able.`